setup.cfg

```diff
@@ -0,0 +1,20 @@
+[metadata]
```

Comment:
This file should not be necessary; you can put all the config in `pyproject.toml`.